lib/msal-common/src/cache/CacheManager.ts

```diff
@@ -239,18 +239,28 @@ export abstract class CacheManager implements ICacheManager {
 
     /**
      * Helper function to retrieve RefreshTokenEntity from cache
-     * @param clientId 
-     * @param account 
+     * @param clientId
+     * @param account
      */
     getRefreshTokenEntity(clientId: string, account: AccountInfo): RefreshTokenEntity | null {
-        const refreshTokenKey: string = CredentialEntity.generateCredentialCacheKey(
-            account.homeAccountId,
-            account.environment,
-            CredentialType.REFRESH_TOKEN,
+        const refreshTokenFilter: CredentialFilter = {
+            homeAccountId: account.homeAccountId,
+            environment: account.environment,
+            credentialType: CredentialType.REFRESH_TOKEN,
             clientId
-        );
+        };
+
+        const credentialCache: CredentialCache = this.getCredentialsFilteredBy(refreshTokenFilter);
+        const refreshTokens = Object.keys(credentialCache.accessTokens).map((key) => credentialCache.refreshTokens[key]);
```

Comment:
This should be refreshTokens right?
```suggestion
        const refreshTokens = Object.keys(credentialCache.refreshTokens).map((key) => credentialCache.refreshTokens[key]);
```

Reply:
Yes, sorry missed a checkin. Done